package.json

```diff
@@ -30,12 +30,11 @@
   },
   "dependencies": {
     "@axe-core/react": "^4.6.0",
-    "@navikt/ds-css": "2.2.0",
+    "@navikt/ds-css": "4.4.2",
     "@navikt/ds-css-internal": "2.2.0",
-    "@navikt/ds-icons": "2.2.0",
```

Comment:
ds-icons har blitt erstattet med aksel-icons som ligger som en dependency i ds-react så man ikke trenger å ha den i package.json

Reply:
Hvordan funker det med at fortsatt vi importerer ikoner fra `ds-icons`?

Reply:
Den ble importert via en annen dependency (ds-react-internal), men legger den tilbake for nå. Tenker jeg kan kjøre egen PR på å oppgradere til aksel-icons i stedet. For nå kan det være fint å vite om at alle komponenter bruker aksel-icons, mens vi importerer fra ds-icons. Sånn jeg har skjønt det skal det ikke være noen endringer på ikoner med unntak av på status-ikonene. Så nå kan vi ha to ulike versjoner av status-ikoner i løsningen, men tenker det går fint fordi jeg kommer til å oppgradere til aksel-icons veldig snart (og det er ikke veldig store endringer) Se: https://aksel.nav.no/produktbloggen/vi-lanserer-ny-ikonpakke